docker/README.md

```diff
@@ -35,10 +35,12 @@ docker-machine start default # Start the docker daemon
 # You might want to stop the machine once you are done coding; it takes up
 # a lot of RAM. Run: docker-machine stop default
 
+Building the images below requires about 6 GB of space for Docker.
+
 # build images
-docker build -t olbase:latest -f docker/Dockerfile.olbase . # 30+ min (Win10Home/Dec 2018)
-docker-compose build web # 10+ min (Win10Home/Dec 2018)
-docker-compose build solr # 5+ min (Win10Home/Dec 2018)
+docker build -t olbase:latest -f docker/Dockerfile.olbase . # ~4 minutes on a 3 yr old Macbook Pro (Dec 2019)
+docker-compose build web # 1min15sec
+docker-compose build solr # 1 minute
```

Comment:
These timings look really short; docker caches things pretty aggressively, so unless you uninstalled everything and started from scratch, my guess would be your computer had cached some of the images (so these timing aren't taking into account e.g. download time).

In general, I think these timings should be quite liberal; I don't want someone to expect something to take 4 minutes and have it take 30+; that sucks :P However expecting something to take 30+ minutes and having it take 4 is much better. I'd recommend _adding_ your timings, like so:

```
docker build -t olbase:latest -f docker/Dockerfile.olbase . # 30+ min (Win10Home/Dec 2018); 5 min (MacBook Pro/Dec 2019)
```

But _only_ if you've done a _full_ reset; these timings are mainly for first-comers.

Reply:
(Also, although a MacBook Pro is a very common device among established developers, my guess would be it's well outside of the price range of students/contributors/etc).